src/components/Instance/Examples/index.svelte

```diff
@@ -0,0 +1,17 @@
+<table>
```

Comment:
Same comment for the format as above: `<div>da title</div><div>the content</div>` is the way to go for the grid to do its magic.